docs/api/window-open.md

```diff
@@ -86,4 +86,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-Set to true after the child window gets closed.
+A Boolean that is set to true after the child window gets closed.
```

Comment:
Returns `true` after the child window gets closed.

Assumption: The TypeScript generator maps `true` / `false` -> Boolean.